### PR TITLE
BGDIINF_SB-1805 & BGDIINF_SB-1849: Show / update profile

### PR DESCRIPTION
**Pull request by vladyslav-tk**
[Test link](https://web-mapviewer.dev.bgdi.ch/bgdiinf_sb-1805/index.html)

### CODE REVIEW

File: src/api/profile.api.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+export const profile = (data) => {
+    return new Promise((resolve, reject) => {
+        if (!data || !data.geom) {
+            const errorMessage = `Geom not provided`
+            log('error', errorMessage)
+            reject(errorMessage)
+        }
+        axios
+            .post(`${API_BASE_URL}rest/services/profile.json`, qs.stringify(data), {
```
Comment (gberaudo) [inline src/api/profile.api.js:14]: Have you tried initializing URLSearchParams with an object?

Comment (vladyslav-tk) [reply src/api/profile.api.js]: We can use URLSearchParams instead of qs. I replaced it

**Review by pakb — COMMENTED** [review]
some minor changes
works great on the test link!

we should be able to merge Guillaume's split test PR soon, you might have some work left when it's the case to rebase...

**Comment by vladyslav-tk** [comment]
@pakb branch rebased on latest dev and updates from comments is done

**Review by pakb — APPROVED** [review]
👍

**Pull request merged**